devices/philips.js

```diff
@@ -90,6 +90,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['LTD012'],
+        model: '5111531H5',
+        vendor: 'Philips',
+        description: 'Garnea downlight',
+        meta: {turnsOffAtBrightness1: true},
+        extend: hueExtend.light_onoff_brightness_colortemp(),
```

Comment:
Please add the color temp as described on https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
Updated, I also have some LDT001 fixtures and so queries and set those as well. They happened to be the same, which may be the case for all of the downlights from Philips/Hue but I only have those two models to confirm.